src/inspector/view/widgets/bendgridcanvas.cpp

```diff
@@ -478,23 +490,44 @@ void BendGridCanvas::drawCurve(QPainter* painter, const QRectF& frameRect)
 
     QPointF lastPoint(0, 0);
     QPen pen = painter->pen();
-    pen.setWidth(3);
+    pen.setWidth(CURVE_LINE_WIDTH);
 
     QColor color(currentTheme.values[ACCENT_COLOR].toString());
     pen.setColor(color);
     painter->setPen(pen);
 
+    QRectF frameRectWithoutBorders = frameRect - QMargins(GRID_LINE_WIDTH, GRID_LINE_WIDTH, GRID_LINE_WIDTH, GRID_LINE_WIDTH);
+    auto constrainToGrid = [&frameRectWithoutBorders](const QPointF& point) {
```

Comment:
I would suggest moving this code into a separate static function rather than a lamba. BendGridCanvas::drawCurve is already pretty huge